fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -3264,6 +3264,9 @@ public class Config extends ConfigBase {
     @ConfField(mutable = true, comment = "Whether enable profile in refreshing materialized view or not by default")
     public static boolean enable_mv_refresh_collect_profile = false;
 
+    @ConfField(mutable = true, comment = "Whether enable adding extra materialized view name logging for better debug")
+    public static boolean enable_mv_refresh_extra_prefix_logging = true;
```

Comment:
need to update FE_Configuration.md

Reply:
can I fix this later in the next pr since I found there are other missing configures too?

Reply:
just take care whatever new configuration this PR added.